scripts/prepdocs.py

```diff
@@ -205,10 +276,19 @@ async def main(strategy: Strategy, credential: AsyncTokenCredential, args: Any):
     parser.add_argument(
         "--tenantid", required=False, help="Optional. Use this to define the Azure directory where to authenticate)"
     )
+    parser.add_argument(
+        "--subscriptionid",
```

Comment:
Why did we need to add this?

Reply:
I see, it's just for the managed identity connection string to the storage account. Please update the help for this arg to make it more obvious